src/app/trajectory/map/map.page.html

```diff
@@ -21,7 +21,6 @@
     (leafletMapReady)="onMapReady($event)"
     (leafletMapMoveEnd)="onMapMoved($event)"
   >
-    <div [leafletLayer]="polyline"></div>
```

Comment:
I'd be great to stay with the "layer management" we are currently using. When there are not performance issues with that, please create a `LayerGroup` (similar to `inferenceHulls`) and store all polylines in this group